dataset/src/test/scala/frameless/functions/NonAggregateFunctionsTests.scala

```diff
@@ -462,6 +461,72 @@ class NonAggregateFunctionsTests extends TypedDatasetSuite {
     check(forAll(prop[Int] _))
   }
 
+  test("inputFileName") {
```

Comment:
Didn't even knew this function exists 👍 . I know that adding them to terget means they should be cleaned up on sbt clean, but I think it would be better if we just delete the temp files on exit. What do you think?

Reply:
Agreed! I'll try to get on this sometime in the next few days. Holidays etc getting in the way.